supervision/detection/core.py

```diff
@@ -729,6 +790,64 @@ def box_area(self) -> np.ndarray:
         """
         return (self.xyxy[:, 3] - self.xyxy[:, 1]) * (self.xyxy[:, 2] - self.xyxy[:, 0])
 
+    def with_nmm(
```

Comment:
This is a very clean implementation and we like it greatly. However, since there's another method we may or may not add, the function API might change, so let's remove this from inference slicer for now. At the same time, it's a good format for a function to be used when testing.